lib/ctblsymm.gi

```diff
@@ -1263,13 +1262,95 @@ InstallMethod( Irr,
     # Compute and store centralizer orders and representative orders.
     if not HasSizesCentralizers( cG ) then
       SetSizesCentralizers( cG,
-          List( cp, x -> gentbl.centralizers[1]( deg, x[2] ) ) );
+          MakeImmutable( List( cp, x -> gentbl.centralizers[1]( deg, x[2] ) ) ) );
```

Comment:
Likewise, as far as I can tell, `SizesCentralizers` is a regular, non-mutable attribute, so this addition of `MakeImmutable` should not have any effect. Do you observe something else (of course there could always be a bug somewhere, or maybe I am missing something and so just wrong)?